.github/workflows/php-ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Wikidata Reference Hunt Game
+
+on: 
+  push:
+    paths:
+      - 'wikidata_game/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        phpVersion: [7.3, 7.4]
```

Comment:
the toolforge php version (and our production as well) is php 7.2. Let's add that too.

Reply:
Really? I thought it was 7.3 according to https://tools.wmflabs.org/phpinfo

In any case, 7.2 is problematic for us since phpunit 9 requires 7.3 or higher.

Reply:
ladsgroup@tools-sgebastion-07:~$ php -v
PHP 7.2.26-1+0~20191218.33+debian9~1.gbpb5a340+wmf1 (cli) (built: Jan  9 2020 12:24:18) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.26-1+0~20191218.33+debian9~1.gbpb5a340+wmf1, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.0, Copyright (c) 2002-2019, by Derick Rethans

Likely the kubernetes webservice is on php7.3 which means it's fine then!